docs/usage.rst

```diff
@@ -105,11 +100,17 @@ BIDS Outputs:
 
 
 Usage:
-""""""
+^^^^^^
 
+.. argparse::
+   :ref: rapidtide.workflows.rapidtide_parser._get_parser
+   :prog: rapidtide
+   :func: _get_parser
 
-    ::
+   Debugging options : @skip
+      skip debugging options
 
+::
       usage: rapidtide in_file outputname [options]
       Perform a RIPTiDe time delay analysis on a dataset.
```

Comment:
I think this section could be removed, since `sphinx-argparse` (called above) can generate the same information automatically.

Reply:
If that works, sure - it will be easier than trying to keep it up to date.